main.go

```diff
@@ -274,6 +299,39 @@ func (k *Kasse) Authenticate(username string, password []byte) (*User, error) {
 	return user, nil
 }
 
+func ResultToLCD(lcd *lcd2usb.Device, res *Result, err error) error {
+	var r, g, b uint8
+	var text string
+	if res != nil {
+		text = res.Display()
+		switch res.Code {
+		default:
+			r, g, b = 255, 255, 255
+		case PaymentMade:
+			r, g, b = 0, 255, 0
+		case LowBalance:
+			r, g, b = 255, 50, 0
+		case AccountEmpty:
+			r, g, b = 255, 0, 0
+		}
+	} else {
+		r, g, b = 255, 0, 0
+		text = err.Error()
+	}
+
+	log.Println(r, g, b)
+	lcd.Color(r, g, b)
+	for i, l := range strings.Split(text, "\n") {
+		lcd.CursorPosition(1, uint8(i+1))
+		fmt.Fprint(lcd, l)
+	}
+	// TODO: Make flag
```

Comment:
(or, alternatively, someone notices that a second is not quite enough to read the display and wants more time. Or everyone thinks, that a second is far too much and people go postal because they have to wait a second before they can buy again)